src/test/java/org/opentripplanner/routing/street/SplitEdgeTurnRestrictionsTest.java

```diff
@@ -151,12 +182,14 @@ private static String computeCarPolyline(Graph graph, GenericLocation from, Gene
     var temporaryVertices = new TemporaryVerticesContainer(graph, request);
     RoutingContext routingContext = new RoutingContext(request, graph, temporaryVertices);
 
-    var gpf = new GraphPathFinder(new Router(graph, RouterConfig.DEFAULT, Metrics.globalRegistry));
+    var gpf = new GraphPathFinder(
+      new Router(graph, transitModel, RouterConfig.DEFAULT, Metrics.globalRegistry)
+    );
     var paths = gpf.graphPathFinderEntryPoint(routingContext);
 
     GraphPathToItineraryMapper graphPathToItineraryMapper = new GraphPathToItineraryMapper(
-      graph.getTimeZone(),
-      new AlertToLegMapper(graph.getTransitAlertService()),
+      transitModel.getTimeZone(),
```

Comment:
Same here. This test routes only on the street network. No transit model needed.

Reply:
Test simplified